python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -123,11 +127,17 @@ class PiffPsfDeterminerConfig(BasePsfDeterminerTask.ConfigClass):
 
     def setDefaults(self):
         super().setDefaults()
-        # stampSize should be at least 25 so that
+        # stampSize for PIFF means the size of input stamps extracted for each
+        # object.
+        #
+        # stampSize should be at least modelSize*sqrt(2)/samplingSize
+        # to ensure that rotated images have support in the model
+        # also we want to ensure
         # i) aperture flux with 12 pixel radius can be compared to PSF flux.
         # ii) fake sources injected to match the 12 pixel aperture flux get
         #     measured correctly
-        self.stampSize = 25
+        # Default modelSize is 25, and 35 is about sqrt(2) * 25
+        self.stampSize = 35
```

Comment:
Given that in the default processing we don't use sky coordinates, I'm inclined to keep this as 25 in the default here. For your testing purposes, you should be able to override this value.

Reply:
@rmjarvis Is there any disadvantage to having `modelSize == stampSize`?

Reply:
Yes.  You definitely don't want stampsize == modelsize.  It should be larger for 3 reasons. 
1. The Lanczos interpolation extends the active modeling space out from the modelsize by interpolant.xrange.  I think your default is Lanczos(11), so it will reach out by 11 pixels on all sided.  That alone suggests you want stampSize = modelSize + 22.
2. Even if your centroids are absolutely perfect, there is a sub-pixel variation of half a pixel in some direction.  So you would want to add one more pixel of data around the edges to make sure you are properly constraining all the pixels around the edge.  That adds another +2 to the stamp size.
3. Finally, the rotation.  This multiplies that calculation by up to sqrt(2). (1/max(|sin(theta)|,|cos(theta)|) if you want to calculate it explicitly for a given WCS.)

And honestly, there is barely any down side to using a larger than needed stamp size.  Piff will just ignore any pixels that are not being used to constrain the model.  So I'd recommend the default being much larger in fact.  Use 80x80 or something.

Reply:
The default I put in was 35 which was designed to account for your part 3. in the list, since the default `modelSize` was set to 25.  It seems like 1. and 2. imply we should go even  larger

Reply:
> The Lanczos interpolation extends the active modeling space out from the modelSize by interpolant.xrange()

Mike, could you confirm that this is needed even if the modeling is happening in pixel coordinates as opposed to sky coordinates?

Reply:
"Needed"?  No. Not technically.  But it will produce inaccurate PSF models near the edges if you don't.

Reply:
Also, I remember the compute scales as cubic power of the size, but does it as `modelSize**3` or `stampSize**3`? I hope the former.

Reply:
modelSize**6 actually.